app/models/like.rb

```diff
@@ -0,0 +1,11 @@
+# Service to download ftp files from the serve
+class Like < ApplicationRecord
+  belongs_to :post, counter_cache: :likes_counter
+  belongs_to :user
+
+  def update_likes_counter
+    post.likes_counter = 0 if post.likes_counter.nil?
+    posts.likes_counter += 1
+    posts.save
+  end
```

Comment:
- Same issue as above. Let the model only take care of increasing the `likes_counter` for a post. Reminder, it's a `post` we are working with, not `posts`. Please refactor as suggested in the above(previous) review thread.

  ### OPTIONAL SUGGESTION
  
  We can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - `after_save` to call the `#update_likes_counter` method every time a new post has been saved to the database, then make the `update_likes_counter` method private.

  ```ruby
    # app/models/like.rb
    after_save :update_likes_counter

    private

    def update_likes_counter
      post.increment!(:likes_counter)
    end
  ```